### PR TITLE
[NEW] Add export as PDF and send via email options

### PR DESCRIPTION
**Pull request by filipemarins**
## Proposed changes (including videos or screenshots)

Add the dropdown menu on chat transcript button.

Screenshot 2022-12-19 at 16 00 43

## Issue(s)

oc-487

## Further comments

Depends on this [PR](https://github.com/RocketChat/fuselage/pull/939).

### CODE REVIEW

File: packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
-		key={id}
-		icon={icon}
-		position='relative'
-		tiny
-		overflow='visible'
-		color={!!color && color}
-		{...{ ...props, ...(tooltip ? { 'data-tooltip': tooltip, 'title': '' } : { title }) }}
-	/>
-);
+const ToolBoxAction: FC<any> = forwardRef(function ToolBoxAction(
```
Comment (aleksandernsilva) [inline packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx:5]: Why do you need the `ref` here?

**Review by aleksandernsilva — CHANGES_REQUESTED** [review]

File: packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx (hunk 1 above)
Comment (filipemarins) [reply packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx:5]: I need the `reference`  to send to the `Dropdown` component to its position correctly.

Comment (filipemarins) [reply packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx:5]: https://github.com/RocketChat/Rocket.Chat/pull/27579/files#diff-818a123e5e5c1d53e9de49e0a9371760a63314456a1c6306c663d8af8c2f300bR32

**Review by aleksandernsilva — APPROVED** [review]

File: packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx (hunk 1 above)
Comment (aleksandernsilva) [reply packages/ui-client/src/components/Header/ToolBox/ToolBoxAction.tsx:5]: got it

**Pull request merged**